unittest/test_diskann.cpp

```diff
@@ -54,8 +54,12 @@ std::string kDir = fs::current_path().string() + "/diskann_test";
 std::string kRawDataPath = kDir + "/raw_data";
 std::string kIpIndexDir = kDir + "/ip_index";
 std::string kL2IndexDir = kDir + "/l2_index";
+std::string kMergedL2IndexDir = kDir + "/merged_l2_index";
+std::string kMergedIpIndexDir = kDir + "/merged_ip_index";
 
-const knowhere::DiskANNBuildConfig build_conf{kRawDataPath, 100, 150, 0.2, 0.2, 4, 0};
+const double merged_build_ram_limit = static_cast<double>(kNumRows * kDim) * sizeof(float) / (1024 * 1024 * 1024) + std::numeric_limits<float>::epsilon();
+const knowhere::DiskANNBuildConfig merged_vamana_build_conf{kRawDataPath, 50, 90, 0.2, merged_build_ram_limit, 4, 0};
```

Comment:
Let's move these into the test function. Since merged index will not be shared with other test cases